### PR TITLE
Run CI on PHP 8.0

### PR DESCRIPTION
**Pull request by szepeviktor**
and name things
and put things in execution order

### CODE REVIEW

File: .github/workflows/wp-phpunit.yml [hunk 1]
```diff
@@ -39,17 +36,17 @@ jobs:
         extensions: mysql
         tools: wp-cli, phpunit:${{ matrix.phpunit-versions }}
 
-    - name: Check PHP Version
-      run: php -v
```
Comment (szepeviktor) [inline .github/workflows/wp-phpunit.yml:43]: PHP version is displayed in the output of `shivammathur/setup-php`

**Comment by szepeviktor** [comment]
We have to wait for WordPress Test Suite to become PHP 8 compatible.

**Comment by szepeviktor** [comment]
There is something in the section _Build & Test Tool Related Changes_ 
https://make.wordpress.org/core/2020/11/23/wordpress-and-php-8-0/
WordPress core developers decided to patch PHPUnit
https://core.trac.wordpress.org/changeset/49037

**Comment by szepeviktor** [comment]
Too esoteric for viktor!

**Comment by afragen** [comment]
Believe me I would love to run the tests on PHP 8. 

Actually I would live to have more tests to run.

**Comment by szepeviktor** [comment]
Very nice lab-level PPE on your profile image!

**Pull request closed without merging**